mod_deploy/controllers.py

```diff
@@ -63,6 +63,9 @@ def decorated_function(*args, **kwargs):
 
 
 def is_valid_signature(x_hub_signature, data, private_key):
+    """
+    Check github deploy signature key (valid or not)
```

Comment:
I'm missing the passed in parameters with their types.

Also, if you add documentation for this file, please do it for the entire file.